manticore/ethereum.py

```diff
@@ -287,21 +287,100 @@ def __mul__(self, reps):
     SUINT = SByte(32)
     SValue = None
 
+    @staticmethod
+    def _tuple_bytes_to_long(tpl):
+        return reduce(lambda acc, b: (acc << 8) + ord(b), tpl, 0L)
+
+    class StaticBytes(object):
+        def __init__(self, value, size=32):
+            if not 0 < size <= 32:
+                raise ValueError('bytes<M>: binary type of M bytes, 0 < M <= 32.')
+            if isinstance(value, (int, long)):
```

Comment:
do you think it's necessary for the ctor to be so "accommodating" and accept so many different types? if this not required, i think i'd prefer to choose one input type/format. 

alternatively, instead of creating a new class for StaticBytes, could we reuse the builtin `bytearray` type and check for that in the big type check in ABI.serialize?